.ci/E2E-tests/execute.sh

```diff
@@ -26,6 +28,8 @@ else
     COMPOSE_FILE="cypress-E2E-tests-postgres.yml"
   elif [ "$CONFIGURATION" = "local" ]; then
     COMPOSE_FILE="cypress-E2E-tests-local.yml"
+  elif [ "$CONFIGURATION" = "multi-node" ]; then
+    COMPOSE_FILE="cypress-E2E-tests-multi-node.yml"
   else
     echo "Invalid configuration. Please choose among mysql, postgres or local."
```

Comment:
If I undestand the code corretly the echo should be updated aswell

```suggestion
    echo "Invalid configuration. Please choose among mysql, postgres, local or multi-node."
```